enos/modules/aws_rdp_server/main.tf

```diff
@@ -0,0 +1,175 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: BUSL-1.1
+
+terraform {
+  required_version = ">= 1.1.2"
+
+  required_providers {
+    enos = {
+      source = "registry.terraform.io/hashicorp-forge/enos"
+    }
+  }
+}
+
+data "enos_environment" "current" {}
+
+data "aws_caller_identity" "current" {}
+
+data "aws_ami" "infra" {
+  most_recent = true
+  owners      = ["amazon"]
+  filter {
+    name     = "name"
+    values   = ["Windows_Server-${var.server_version}-English-Full-Base*"]
```

Comment:
Another `enos_fmt` pr status check

To make it easier, there should be a make target in the enos directory
```
make fmt
```